third_party/subscriptions-project/swg.js

```diff
@@ -6727,11 +6727,12 @@ class EntitlementsManager {
   }
 
   /**
+   * @param {?string} opt_encryptedDocumentKey
```

Comment:
Can't push changes in the `third_party` outside of a release process. Please move changes to the swg-js project and revert them here.

Reply:
done